### PR TITLE
accession web archive seed as part of accession_web_archive_spec

### LINKED ISSUE #14: Web archiving seed objects need integration tests
- accessioning seed into SDR for the first time (would this be via was-registrar-app?  via pre-assembly?  both??)
- updating existing SDR web archival seed object

The test should mimic what is done for was seed  object (e.g. start in was-registrar-app?) and should check for was crawl object-ness in Argo. We would like to run something through the was robots if we ca/

Want to test all the way through accessioning (was-robot-suite and common-accessioning) if that is possible, and definitely check for was seed object-ness in Argo.

Talk to Andrew, and ask in web-archiving slack channel.

### PR DESCRIPTION
**Pull request by ndushay**
Hold for @lwrubel or @edsu or @andrewjbtw or @peterchanws  to sign off

## Why was this change made? 🤔

Closes #14

we needed an integration test for web archiving seed objects.  At Ed's suggestion, I folded it into the crawl accession test so the crawl itself can be known to exist.

### CODE REVIEW

File: spec/features/accession_web_archive_spec.rb [hunk 1]
```diff
@@ -20,7 +22,7 @@
       file.close
     end
   end
-  let(:warc_path) { 'spec/fixtures/data.warc' }
```
Comment (ndushay) [inline spec/features/accession_web_archive_spec.rb:23]: moved up

File: spec/features/accession_web_archive_spec.rb [hunk 2]
```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
-RSpec.describe 'Use was-registrar-app, Argo, and pywb to ensure web archives are accessioning', type: :feature do
+RSpec.describe 'Use was-registrar-app, Argo, and pywb to ensure web archive crawl and seed accession', type: :feature do
   let(:job_specific_directory) { start_time.to_i.to_s }
   let(:remote_path) do
     "#{Settings.was_registrar.username}@#{Settings.was_registrar.host}:#{Settings.was_registrar.jobs_directory}/" \
       "#{job_specific_directory}/"
   end
-  let(:source_id) { "test:#{start_time.to_i}" }
```
Comment (ndushay) [inline spec/features/accession_web_archive_spec.rb:9]: moved down

**Review by aaron-collier — APPROVED** [review]

**Review by justinlittman — APPROVED** [review]

**Comment by ndushay** [comment]
Actually I'm going to merge this as is, but I still want one of @lwrubel @andrewjbtw @edsu or @peterchanws  to sign off on this as testing what is needed for was-seed accessioning.

I wish it were simple to wait for the purl page to show the sul-embed viewer ... but that is not easy, and my judgement was that looking at the purl public xml was not useful in this case -- I think the test as written already verifies the thumbnail image for the seed.

**Comment by ndushay** [comment]
The need to sign off on this has been met as Andrew and I spoke.  I created issue #456 as a result of that conversation.  Whee!

**Pull request merged**